reactor-netty-http/src/test/java/reactor/netty/http/server/logging/error/ErrorLogTest.java

```diff
@@ -0,0 +1,197 @@
+/*
+ * Copyright (c) 2025 VMware, Inc. or its affiliates, All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package reactor.netty.http.server.logging.error;
+
+import ch.qos.logback.classic.Logger;
+import ch.qos.logback.classic.spi.ILoggingEvent;
+import ch.qos.logback.classic.spi.LoggingEvent;
+import ch.qos.logback.core.Appender;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Mockito;
+import org.slf4j.LoggerFactory;
+import reactor.netty.BaseHttpTest;
+
+import java.time.Duration;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static reactor.netty.http.server.logging.error.ErrorLog.LOGGER;
+
+/**
+ * This test class verifies {@link DefaultErrorLogHandler}.
+ *
+ * @author raccoonback
+ */
+class ErrorLogTest extends BaseHttpTest {
```

Comment:
I would like to check all three HTTP versions: 1.1, 2 and 3
May be instead of separate test class we can make something similar to
reactor.netty.http.HttpProtocolsTests#testAccessLog
reactor.netty.http.HttpProtocolsTests#testAccessLogWithForwardedHeader
reactor.netty.http.Http3Tests#testAccessLog
reactor.netty.http.Http3Tests#testAccessLogWithForwardedHeader

Reply:
@violetagg 
That's a great point.
I will reflect it!